app/models/service_template.rb

```diff
@@ -356,4 +361,27 @@ def create_resource_actions(ae_endpoints)
     end
     save!
   end
+
+  def self.create_from_options(options)
+    create(options.except(:config_info).merge(:options => { :config_info => options[:config_info] }))
+  end
+  private_class_method :create_from_options
+
+  private
+
+  def construct_config_info
+    config_info = {}
+    if service_resources.where(:resource_type => 'MiqRequest').exists?
+      config_info.merge!(service_resources.find_by(:resource_type => 'MiqRequest').resource.options.compact)
+    end
+
+    resource_actions.each do |resource_action|
```

Comment:
@jntullo You may need to move this part into a separate method. Orchestration and AnsibleTower need their own `construct_config_info` to find the template and manager from `service_resources`. Again retrieving IDs should be sufficient.